.python-version

```diff
@@ -0,0 +1 @@
+2.7.14
```

Comment:
Why do we need python?

Reply:
i have NO IDEA, but something in the `kubectl` code we're executing relies on it:
```Unable to connect to the server: error executing access token command "/Users/maia/google-cloud-sdk/bin/gcloud config config-helper --format=json": err=exit status 127 output= stderr=pyenv: python2: command not found

The `python2' command exists in these Python versions:
  2.7.14```

however, i don't think pyenv needs to care about it unless your pyenv happens to be borked from the last project (like mine). fixed and removed this file.